<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>[git] Visual Branch Diff</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>GIT directory:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="locationEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="locationPick">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="branch1Label">
      <property name="text">
       <string>From (branch1):</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QComboBox" name="branch1Combo">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="branch1Color">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="branch2Label">
      <property name="text">
       <string>To (branch2):</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QComboBox" name="branch2Combo">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="branch2Color">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Output:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="typesBox">
      <item>
       <property name="text">
        <string>PNG Image</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Scalable Vector Graphics</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Portable Document Format</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Display:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="showEdgeDiff">
        <property name="text">
         <string>Show SHA1 on edges</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showEdgeWeight">
        <property name="text">
         <string>Show +/- on edges</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="runButton">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="runProgress">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>showEdgeDiff</sender>
   <signal>toggled(bool)</signal>
   <receiver>showEdgeWeight</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
